### DISCUSSION TITLE
Is it possible to create query directives ?

### DISCUSSION THREAD

**Discussion by Tiross** [mirumee/ariadne#506] [Q&A] [answered]
Hello, 

The documentation only speaks about schema directive, I would like to add query directives to allow the API consumer to allow them some query customization. 

I would like to have some thing like (based on swapi data):
```graphql
{
  allFilms {
    films {
      en: title @translate(lang: "en")
      fr: title @translate(lang: "fr")
      it: title @translate(lang: "it")
    }
  }
}
```

And obtains the directive data during query to use it. 

Is it possible ?

**Comment by rafalp** [comment, marked as answer]
Ariadne doesn't support custom query directives. Why you can't use field argument for your case?

```graphql
{
  allFilms {
    films {
      en: title(lang: "en")
      fr: title(lang: "fr")
      it: title(lang: "it")
    }
  }
}
```

**Reply by Tiross** [reply]
Mainly because I forgot I can do it.... 

Sorry and thank you
